shared/src/commonMain/resources/MR/colors/colors.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- format: #RRGGBB[AA] or 0xRRGGBB[AA] or RRGGBB[AA] where [AA] - optional -->
+    <color name="primary">
```

Comment:
Let's discuss if we need these colors here or if they can just live in common colors.kt file.